test-network-function/networking/suite.go

```diff
@@ -503,10 +501,11 @@ func testListenAndDeclared(env *config.TestEnvironment) {
 			err = listeningPortList(commandlisten, nodeOc.Context, listeningPorts)
 			if err != nil {
 				tnf.ClaimFilePrintf("Failed to get listening port for pod name %s in pod namespace %s due to %v, skipping this pod", podUnderTest.Name, podUnderTest.Namespace, err)
+				skippedPods = append(skippedPods, *podUnderTest)
 				continue
 			}
 			// compare between declaredPort,listeningPort
-			undeclaredPorts = checkIfListenIsDeclared(listeningPorts, declaredPorts)
+			undeclaredPorts := checkIfListenIsDeclared(listeningPorts, declaredPorts)
 			for k := range undeclaredPorts {
 				tnf.ClaimFilePrintf("The port %d on protocol %s in pod name %s and pod namespace is %s not declared.", k.port, k.protocol, podUnderTest.Name, podUnderTest.Namespace)
```

Comment:
In order to make this trace easier to understand for the customer when this test case fails, I suggest this change:
```
tnf.ClaimFilePrintf("pod %s ns %s is listening on port %d protocol %d, but that port was not declared in any container spec.",
    podUnderTest.Name, podUnderTest.Namespace, k.port, k.protocol)
)
```
As there's no way (yet) to know which container of the pod actually opened that port, that's the most reasonable message we can print. What do you think, @edcdavid & @sebrandon1 ?

Reply:
I agree.

Reply:
I agree too